src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs

```diff
@@ -17,8 +19,11 @@ public static class DicomElementMinimumValidation
         private static readonly Regex ValidIdentifierCharactersFormat = new Regex("^[0-9\\.]*$", RegexOptions.Compiled);
         private const string DateFormat = "yyyyMMdd";
 
-        public static void ValidateCS(string value, string name)
+        public static void ValidateCS(DicomElement element)
         {
+            EnsureArg.IsNotNull(element, nameof(element));
+            string name = GetName(element.Tag);
```

Comment:
If you're calling this to throw an exception for a particular case, and you're using quite a lot in the class, I'd change this method (or create a new one) for that purpose explicitly. For example, often `IDisposable` classes have a method called `ThrowIfDisposed()` whose entire purpose is to throw `IObjectDisposedException` if the class has already been disposed.